.claude/hooks/pre_tool_use.py

```
@@ -0,0 +1,142 @@
+#!/usr/bin/env -S uv run --script
```

Ruff (0.12.2)

Shebang is present but file is not executable

(EXE001)